classwork 4.10/classwork 4.10/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace hometask2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            double V;
```

Comment:
Локальные переменные принято называть с маленькой буквы. И лучше использовать не однобуквенные, а значимые имена.
```suggestion
            double volume;
```